docs/src/pages/components/dropdown.mdx

```diff
@@ -381,7 +360,7 @@ export default Example
   <Types.Prop name="error" type="boolean" desc="error state flag"/>
   <Types.Prop name="label" type="string" desc="identifying string for dropdown" />
   <Types.Prop name="menu" type="Dropdown.Item(s)" desc="menu items for dropdown" default="<span></span>" />
-  <Types.Prop name="onChange" type="function" desc="triggered when an item selected ((event, value, label) => ())" />
+  <Types.Prop name="onChange" type="(Event, value, label) => ()" desc="triggered when an item selected ((event, value, label) => ())" />
```

Comment:
```suggestion
  <Types.Prop name="onChange" type="(Event, value, label) => void" desc="triggered when an item selected" />
```

Just fyi, the old docs use `()` a fair amount in return types.  This is correct in flowtype syntax.  For TS, return types are instead `void`.